pkg/scheduler/algorithm/predicates/metadata.go

```diff
@@ -330,20 +331,31 @@ func RegisterPredicateMetadataProducerWithExtendedResourceOptions(ignoredExtende
 }
 
 // GetPredicateMetadata returns the predicateMetadata which will be used by various predicates.
-func GetPredicateMetadata(pod *v1.Pod, nodeNameToInfoMap map[string]*schedulernodeinfo.NodeInfo) PredicateMetadata {
+func GetPredicateMetadata(pod *v1.Pod, sharedLister schedulerlisters.SharedLister) PredicateMetadata {
```

Comment:
It seems be enough to use `NodeInfoLister` instead of `sharedLister`?

Reply:
perhaps for now, may be not in the future.